src/api.mli

```diff
@@ -150,17 +150,20 @@ module Decoders : functor (D : Decoders.Decode.S) -> sig
   module Request : sig
     module Hints : sig
       module Induct : sig
-        val structural : (D.value, Request.Hints.Induct.t) D.t_let
+        (* val structural : (D.value, Request.Hints.Induct.t) D.t_let *)
+        val structural : Request.Hints.Induct.t D.decoder
 
-        val functional : (D.value, Request.Hints.Induct.t) D.t_let
+        (* val functional : (D.value, Request.Hints.Induct.t) D.t_let *)
```

Comment:
Any reason to leave the old code in as comments? If not, can we just get rid of it?

Reply:
Done